test/gameplay_test.rb

```diff
@@ -25,35 +25,34 @@ def test_it_exists
     assert_instance_of Gameplay, game
   end
 
-  # def test_player_start
```

Comment:
Nice work making it so that we can have only one boat. Now the computer might have a chance